src/imitation/data/rollout.py

```diff
@@ -596,8 +594,54 @@ def rollout_and_save(
     if verbose:
         stats = rollout_stats(trajs)
         logging.info(f"Rollout stats: {stats}")
+    return trajs
+
+
+def rollout_and_save(
```

Comment:
I think `rollout_and_save` is only used in `train_rl`, so I'm tempted to just remove this function given it's basically a one-liner anyway

Reply:
Good point. I removed it.